tests/python/unittest/test_operator.py

```diff
@@ -9337,4 +9337,82 @@ def test_elemwise_sum_for_gradient_accumulation():
         assert stored_grad['write'] == stored_grad['add']
         assert stored_grad['write'] == 2 * nrepeat
 
+@with_seed()
+def test_elementwise_ops_on_misaligned_input():
```

Comment:
it would be great to have the numpy equivalent of them too, in tests/python/unittest/test_numpy_op.py

Reply:
:+1: